include/old/test.h

```diff
@@ -44,6 +44,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdint.h>
+#ifndef __GLIBC__
+#include <bits/reg.h>
+#endif
 
```

Comment:
That is quite ugly fix for the problem, there is no guarantee that the file exist on non-glibc target (there are more libc implementation than just musl and glibc).

Unfortunately the `__WORDSIZE` appears to be standard way to tell if we are compiling with 64bit or 32bit ABI.

Maybe we can just add a fallback definition into the LTP to the `include/lapi/abisize.h` and include it when needed, something as:

```
#ifndef __WORDSIZE
# define __WORDSIZE (sizeof(void*) == 8 ? 64 : 32)
#endif /* __WORDSIZE */
```

But I'm not sure that this is correct for the x32 ABI and similar oddities.